lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1627,7 +1627,7 @@ namespace Js
             // We do not own the memory passed into DefaultLoadScriptUtf8. We need to save it so we copy the memory.
             if (*ppSourceInfo == nullptr)
             {
-                *ppSourceInfo = Utf8SourceInfo::New(this, script, parser->GetSourceIchLim(), cb, pSrcInfo, isLibraryCode);
```

Comment:
It looks like this is a bug - parse->GetSourceIchLim was returning random numbers. may be module was not doing something right to have that? @Yongqu - do you know about this ? the line below we use the length but here we were using GetSourcveIchLim.